BinaryFile.py

```diff
@@ -0,0 +1,41 @@
+import UserManagement
+import os
+import imghdr
```

Comment:
I think you missed a large point in this project, you have not used any inheritance methods. PLease go over the notebook again and correct your files.
Furthermore, please sort out your, commits some of the files are empty and it isn't clear what shoudl be reviewed.